src/main/java/com/hltech/pact/gen/domain/client/annotation/handlers/DeleteMappingMethodsHandler.java

```diff
@@ -25,19 +27,36 @@ public boolean isSupported(Method method) {
     }
 
     @Override
-    public RequestRepresentation handle(Method method) {
+    public RequestRepresentation handleRequest(Method method) {
         return RequestRepresentation.builder()
             .httpMethod(HttpMethod.DELETE)
             .path(getPathFromMethod(method))
             .headers(combineHeaders(
-                method.getAnnotation(DeleteMapping.class).headers(),
+                ArrayUtils.addAll(method.getAnnotation(DeleteMapping.class).headers(), getRequestMediaHeaders(method)),
                 RequestHeaderParamsExtractor.extractAll(method)))
             .body(RequestBodyExtractor.extract(method.getParameters()))
             .requestParameters(RequestParametersExtractor.extractAll(method))
             .pathParameters(PathParametersExtractor.extractAll(method))
             .build();
     }
 
+    @Override
+    public String[] getResponseMediaHeaders(Method method) {
+        return Arrays.stream(method.getAnnotation(DeleteMapping.class).produces())
+            .map(header -> "Content-Type=" + header)
```

Comment:
Please move string "Content-Type=" to variable

Reply:
It id hidden in the private method.

Reply:
Used predefined constant